lib/chef/knife/mixin/helper.rb

```diff
@@ -59,7 +59,7 @@ def validate_json(json)
             next unless printable?(value.to_s)
 
             msg = "Value '#{value}' of key '#{key}' contains non-printable characters. Check that backslashes are escaped with another backslash (e.g. C:\\\\Windows) in double-quoted strings."
-            raise ChefVault::Exceptions::InvalidValue, msg
+            ChefVault::Log.warn(msg)
```

Comment:
i believe an Exception still needs to be raised as before, except that the allowed characters have expanded.